cypress/integration/common/steps.js

```diff
@@ -1,4 +1,8 @@
-import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+import {
+  Given,
+  When,
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T09:16:07Z" title="Friday, August 30th 2019, 11:16:07 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,7 @@
 import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
 import { getLangByName } from "../../support/helpers";
 import slugify from "slug";
+// import { VERSION } from '../../../webapp/pages/terms-and-conditions.vue';
```


I would use this import to avoid future fails of the build. 🍀
… if the version is increased.